### DISCUSSION TITLE
make support camera (mxiang.camera.mwc10 + mxiang.camera.mwc11)

### DISCUSSION THREAD

**Discussion by fedyagrib** [rytilahti/python-miio#1821] [Q&A]
hello!

I found your library and decided to make support of my camera because it can't make continues recording. so I've got json for both receiver and camera but I can't understand how to access to camera via receiver because receiver doesn't have some nested command like access_to_camera[camera_token].camera_function. I will be really glad if you help with that

**Comment by laheller** [comment]
Following.

**Comment by laheller** [comment]
Hi @fedyagrib 

Do you have any progress on this?
My problem is that I would like to display the camera live image on my notebook rather than on Android device.

BR,

Ladislav
